src/main/java/org/myrobotlab/service/meta/LocalSpeechMeta.java

```diff
@@ -20,7 +20,8 @@ public LocalSpeechMeta(String name) {
     addDescription("Local OS text to speech ( tts.exe / say etc ... )");
     setAvailable(true);
     addCategory("speech");
-    addDependency("com.microsoft", "tts", "1.1", "zip");
+    // addDependency("com.microsoft", "tts", "1.1", "zip");
```

Comment:
how do the binaries get installed?

Reply:
tts.exe is a tiny executable to access ms SAPI system - which is where the implementation is..
I bundled tts.exe in the resource of LocalSpeech

Reply:
mimic sounds like crap on windows - is there any real reason to continue with it on ms os ?